iot_config/src/region_map.rs

```diff
@@ -127,21 +127,21 @@ pub async fn update_region(
         on conflict (region) do update set params = excluded.params
         "#,
     )
-    .bind(region)
+    .bind(region.to_string())
     .bind(params.encode_to_vec())
     .execute(&mut transaction)
     .await?;
 
     let updated_region = if let Some(indexes) = indexes {
         sqlx::query(
             r#"
-            insert into regions (region, indexes)
-            values ($1, $2)
-            on conflict (region) do update set indexes = excluded.indexes
+            update regions
```

Comment:
is it not possible to include the indexes in the prior upsert ?

Reply:
i wasn't optimistic this was possible since the indexes are a nullable field but as it turns out this _is_ possible thanks to some conditional logic in the query and the fact that sqlx will automatically accept type `T` or `Option<T>` interchangeably when dealing with a nullable field (TIL!); thanks for the push

Reply:
[8a48ab7](https://github.com/helium/oracles/pull/348/commits/8a48ab7d9556143bf60d1255a544ff0e754185bb)